src/app/fyle/split-expense/split-expense.page.ts

```
@@ -127,6 +127,8 @@
 
   unspecifiedCategory: OrgCategory = null;
 
+  isReviewModalOpen = false;
+  
```

ESLint

[error] Trailing spaces not allowed.

(no-trailing-spaces)